src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -256,24 +259,15 @@ public Command setElvPercent(double percentPower) {
         });
   }
 
-  private void disableSoftLimits() {
-    elevatorMotor1
-        .getConfigurator()
-        .apply(
-            new SoftwareLimitSwitchConfigs()
-                .withForwardSoftLimitEnable(false)
-                .withReverseSoftLimitEnable(false));
-  }
-
-  private void enableSoftLimits() {
-    elevatorMotor1.getConfigurator().apply(elvSoftLimit);
-  }
-
   public Command limitHit() {
-    return runOnce(this::disableSoftLimits)
+    return runOnce(
+            () -> {
+              disableSoftLimits();
+            })
```

Comment:
Original is more concise

Reply:
honestly don't know why that got changed